README.md

```diff
@@ -1,2 +1,32 @@
-# SSS
-Database Code
+# Self-Sufficiency Standard Database
+
+## Background
+[The Self-Sufficiency Standard](https://selfsufficiencystandard.org/)(SSS) was created by the Center of Women's Welfare (CWW) at the University of Washington as an alterantive to the Official Poverty Measure (OPM). The Self-Sufficiency Standard data is spread across the CWW website and this repository creates a database to hold the Self-Sufficiency Standard data. 
+
+## Setting Up Computer
+It's important to set up your machine these [instructions](https://neurohackademy.org/setup/).
+
+## Cloning the Repository
+With your machine set up, we recommend working with your terminal to clone the repository. All the following instructions will be ran exclusively in the terminal. 
+
+First, you would want to choose the directory that is hosting the code. You can use `ls` to list the folders in the current directory and then use `cd [directory_path]` to enter the directory you want to host the repository.
```

Comment:
maybe "choose the directory that is hosting the code" -> "choose the directory where you want the code to be located on your computer". Trying to make it more obvious that this is a choice the user can make.